Model/ConfigProvider/Predefined.php

```diff
@@ -52,4 +52,53 @@ public function getConfig($store = null)
         ];
         return $config;
     }
+
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getLocationLiveWeb($store = null)
+    {
+        return $this->scopeConfig->getValue(
+            static::XPATH_PREDEFINED_LOCATION_LIVE_WEB,
+            ScopeInterface::SCOPE_STORE,
+            $store
+        );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getLocationTestWeb($store = null)
+    {
+        return $this->scopeConfig->getValue(
+            static::XPATH_PREDEFINED_LOCATION_TEST_WEB,
+            ScopeInterface::SCOPE_STORE,
+            $store
+        );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getWsdlLiveWeb($store = null)
+    {
+        return $this->scopeConfig->getValue(
+            static::XPATH_PREDEFINED_WSDL_LIVE_WEB,
+            ScopeInterface::SCOPE_STORE,
+            $store
+        );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getWsdlTestWeb($store = null)
```

Comment:
I think we should remove this class and references since we don't use SOAP anymore.

Reply:
"You Good", I will remove it